src/main/webapp/gameStage.css

```diff
@@ -0,0 +1,458 @@
+  color: white;
+  background-color:var(--orange);
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+}
+
+.step-team-button:hover {
+  background-color: #555; 
+}
+
+/** Responsive media classes to addapt to diffrent devices **/
+@media screen and (max-width: 650px) {
+  .step-team-column {
+    width: 100%;
+    display: block;
+  }
+}
+
+@media screen and (max-width: 800px) {
+  /** start of large tablet styles **/
+  .step-column {
+    -ms-flex: 60%;
+    flex: 60%;
+    max-width: 60%;
+  }
+
+  .step-caption spam.border {
+    font-size: 20px;
+  }
+  
+  h1 {
+    font-size: 4.4rem;
+  }
+
+  h2 {
+    font-size: 1.4rem;
+  }
+
+  h3 {
+    font-size: 1.0rem;  
+  }
+
+  h4 {
+    font-size: 0.70rem;
+  }
+
+  p {
+    font-size: 1.0rem;
+    line-height: 1.6rem;
+  }
+}
+
+@media screen and (max-width: 479px) {
+  /** start of phone styles **/
+  .step-column {
+    -ms-flex: 100%;
+    flex: 100%;
+    max-width: 100%;
+  }
+
+  .step-caption spam.border {
+    font-size: 14px;
+  }
+
+  h1 {
+    font-size: 2.4rem;
+  }
+
+  h2 {
+    font-size: 1.2rem;
+  }
+
+  h3 {
+    font-size: 0.95rem;  
+  }
+
+  p {
+    font-size: 0.96rem;
+    line-height: 1.0rem;
+  }
+}
+
+/** classes for the dialog container **/
+.step-dialogue {
+  text-align: left;
+  margin-left: 100px;
+  margin-right:100px;
+  padding: 16px;
+  border: 5px solid var(--orange);
+  background-color: rgba(0, 0, 0, 0.3)
+}
+
+/** Player Sprite **/
+#player-picture {
+  object-fit: cover;
+  position: absolute;
+  border-radius: 50%;
+  left : 50px;
+  width : 170px;
+  height : 185px;
+}
+
+#pixel-avatar {
+  position: absolute;
+  width : 260px;
+  left : 10px;
+  top : 160px;
+  height : 260px;
+}
+
+#image-container {
+  position: absolute;
+  height : 550px;
+  left : 70%;
+  top : 250px;
+  float : left;
+}
+
+#promotion-button {
+  margin-top: 10px;
+  margin-right: 10px;
+  position:absolute;
+  top:0;
+  right:0;
+}
+
+/** Quiz Pages **/
+#quiz-container {
+  padding-top : 5px;
+  padding-bottom : 20px;
+  background-color: var(--orange);
+  border-radius: 8px;
+  border-style: solid;
+  border-color: var(--yellow);
+  width: 70%;
+  margin: auto;
+}
+
+#question-text {
+  font-weight: 900;
+  color: var(--charcoal);
+}
+
+#choices-container {
+  text-align: left;
+  width: 70%;
+  margin: auto;
+}
+
+/** promotion quiz classes **/
+.step-meme img {
+  border: 5px solid var(--orange);
+  width: 600px;
+  height: 630px; 
+}
+
+.step-yellow {
+  color: var(--yellow);
+}
+
+/** imageHandler classes **/
+.step-textarea {
+  border: 2px solid var(--orange);
+}
+
+.step-oneline {
+  display:flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+}
+
+.step-center-imagehandler {
+  position:absolute;
+  margin-left: 39%;
+  margin-right: 39%;
+}
+
+/* classes for music player */
+.align-bottom-center{
+    position: absolute;
+    margin-left: 50%;
+    margin-right: 50%;
+    margin-top: 25%;
+}
+
+audio {
+float: left;
+}
+
```

Comment:
It seems pretty Identical to style.css except for the last 10 lines. If they're going to be two separate files, there should probably be a greater difference between the two 450 line files or just the one style sheet

Reply:
+1